indy_client/test/cli/test_node_demotion.py

```diff
@@ -1,51 +0,0 @@
-import pytest
-from plenum.common.signer_did import DidSigner
-from plenum.common.txn_util import get_payload_data
-from stp_core.crypto.util import randomSeed
-from indy_client.test.cli.helper import addAgent
-from plenum.common.constants import SERVICES, VALIDATOR, TARGET_NYM, DATA
-from indy_client.test.cli.constants import NODE_REQUEST_COMPLETED, NODE_REQUEST_FAILED
-
-
-def ensurePoolIsOperable(be, do, cli):
```

Comment:
This tests need to be migrated to SDK and moved to indy-node folder

Reply:
I think we have pretty same functionality in 
indy-plenum/plenum/test/pool_transactions/test_suspend_node.py
Please, tell me, if we need to integrate this in node